### PR TITLE
feat: Okta Fastpass rebranding in SIW

### PR DESCRIPTION
**Pull request by jasleennuno-okta**
## Description:
- feat: Okta Fastpass rebranding in SIW
- includes rewording for Fastpass and centering sign in with fastpass button
- OV icon updates were already done a couple of months ago
- based on the following mocks: https://www.figma.com/file/wra4gnCX3Xj7HnwCFNXQ0A/Okta-FastPass-WIP?node-id=678%3A0

## PR Checklist

- [x] Have you verified the basic functionality for this change?

- [x] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Screenshot:
Screen Shot 2021-07-01 at 12 50 06 PM
Screen Shot 2021-07-01 at 11 11 10 AM
Screen Shot 2021-07-01 at 11 09 18 AM
Screen Shot 2021-07-01 at 11 15 11 AM

### Reviewers:
- @magizh-okta @gowthamidommety-okta @stevennguyen-okta @panchamyadav-okta 

### Issue:

- [OKTA-407172](https://oktainc.atlassian.net/browse/OKTA-407172)

### CODE REVIEW

**Comment by stevennguyen-okta** [comment]
@panchamyadav-okta

File: assets/sass/v2/_identify.scss [hunk 1]
```diff
@@ -33,16 +34,19 @@
       margin-bottom: 0.83em;
     }
     .button {
```
Comment (magizh-okta) [inline assets/sass/v2/_identify.scss:36]: the selectors here are weird here I guess we should get rid of sign-in-with-idp and custom-buttons. cc @jhe-okta 

https://github.com/okta/okta-signin-widget/pull/2083/files#diff-0e0a6554c5f0823c8b023835c2ba22e6fd6a66496d4534cc46c29a88943c3824R30

why would we have a okta-verify-container inside the sign-in-with-idp div.

File: assets/sass/v2/_identify.scss [hunk 2]
```diff
@@ -33,16 +34,19 @@
...
       position: relative;
       min-height: 50px;
       height: auto;
-      padding-top: 10px;
+      padding-top: 15px;
       padding-bottom: 10px;
       line-height: 1.4;
       align-items: center;
       .okta-verify-authenticator {
         flex-shrink: 0;
```
[the remark this answers is not in the document]
Comment (magizh-okta) [reply assets/sass/v2/_identify.scss:46]: Also we have the styles for icon here 
https://github.com/okta/okta-signin-widget/blob/c6f76cb733a38dfea7023fb74ebebfe15d8d8d4a/assets/sass/modules/_btns.scss#L105

would be good to consolidate everything in one are.

File: src/v2/view-builder/views/AutoRedirectView.js [hunk 3]
```diff
@@ -1,30 +1,11 @@
...
-    const appName = appInstanceName ? appInstanceName : appDisplayName;
-
-    if (appName && userEmail) {
-      titleString = loc('oie.success.text.signingIn.with.appName.and.identifier', 'login', [appName, userEmail]);
-    } else if (appName) {
-      titleString = loc('oie.success.text.signingIn.with.appName', 'login', [appName]);
-    }
-
-    return titleString;
+    return loc('oie.success.text.signingIn.with.fastpass', 'login');
```
Comment (magizh-okta) [inline src/v2/view-builder/views/AutoRedirectView.js:6]: we seem to be removing some useful info here. whats the reasoning ?

Comment (jasleennuno-okta) [reply src/v2/view-builder/views/AutoRedirectView.js:6]: @magizh-okta it's from the mocks here - https://www.figma.com/file/wra4gnCX3Xj7HnwCFNXQ0A/Okta-FastPass-(WIP)?node-id=464%3A22851

i'll let @panchamyadav-okta answer this

File: assets/sass/modules/_btns.scss [hunk 4]
```diff
@@ -105,12 +105,12 @@ $od-secondary-button-action-text-color: #124a94;
 .okta-verify-authenticator {
```
Comment (magizh-okta) [inline assets/sass/modules/_btns.scss:105]: @jasleennuno-okta  encapsulate this within .button just in case.

**Comment by panchamyadav-okta** [comment]
@jasleennuno-okta these look great! :) 
Can the text on all of the screens be updated to "Okta FastPass" with an uppercase P?

File: assets/sass/v2/_identify.scss (hunk 1 above)
Comment (jhe-okta) [reply assets/sass/v2/_identify.scss:33]: I don't remember the context. But usually, it is because the change only applies to a certain UI.

Comment (magizh-okta) [reply assets/sass/v2/_identify.scss:33]: @jasleennuno-okta  just realized that https://github.com/okta/okta-signin-widget/pull/2083/files#diff-0e0a6554c5f0823c8b023835c2ba22e6fd6a66496d4534cc46c29a88943c3824R44 

needs those classes at the top to be encapsulated. Ignore my comment.

**Review by panchamyadav-okta — COMMENTED** [review]
We're good to go with `Sign into {app} as {user}`instead of `Signing in with Okta FastPass` as originally proposed.

File: packages/@okta/i18n/src/properties/login.properties [hunk 5]
```diff
@@ -533,7 +533,7 @@ primaryauth.submit = Sign In
 primaryauth.newUser.tooltip=This is the first time you are connecting to {0} from this browser
 primaryauth.newUser.tooltip.close=Close
 oktaVerify.description = To access this resource, your organization requires you to sign in using your device.
-oktaVerify.button = Sign in using Okta Verify on this device
+oktaVerify.button = Sign in with Okta FastPass
```
Comment (magizh-okta) [inline packages/@okta/i18n/src/properties/login.properties:536]: Nit I guess it is better to rename the key to oie.oktaVerify.fastpass.button so developers dont confuse with oktaVerify in v1 or elsewhere.

**Review by magizh-okta — APPROVED** [review]
just a minor suggestion with the key to avoid confusion.

**Review by magizh-okta — APPROVED** [review]

**Pull request closed without merging**